src/util.js

```diff
@@ -1,14 +1,13 @@
 const closest = function(el, selector, rootNode) {
-  const rootElement = rootNode || document.body;
   let element = el;
   const matchesSelector = element.matches
                           || element.webkitMatchesSelector
                           || element.mozMatchesSelector
                           || element.msMatchesSelector;
   while (element) {
-    const flagRoot = element === rootElement;
-    if (flagRoot || matchesSelector.call(element, selector)) {
-      if (flagRoot) {
+    const isRoot = element === rootNode || element.tagName === 'HTML';
```

Comment:
In theory, portals can be put anywhere under html (even outside body tag), so checking html tag here instead of body tag.

Reply:
Suggested replacement with:

``` javascript
const isRoot = element === rootNode || element === document.body;
```

Reply:
updated.